daemon/execdriver/driver.go

```diff
@@ -27,7 +27,7 @@ var (
 // StartCallback defines a callback function.
 // It's used by 'Run' and 'Exec', does some work in parent process
 // after child process is started.
-type StartCallback func(*ProcessConfig, int)
+type StartCallback func(*ProcessConfig, int, <-chan struct{})
```

Comment:
Can you document the channel passed here? Something about being a notifier for oom